packages/interop-tests/src/__test__/interop.test.ts

```diff
@@ -251,17 +255,9 @@ it('browser wallet creates channel + cooperates with server wallet to fund chann
   const serverOutput1 = await serverWallet.joinChannel({channelId});
   await browserWallet.pushMessage(serverMessageToBrowserMessage(serverOutput1), 'dummyDomain');
 
-  /** This is fragile. We are waiting for the third channelUpdated event. Note that these events consistently arrive in the following order.
-   *  But the events are not guaranteed to arrive in this order:
-   *  1. The first event is triggered by the registration of the channel with the chain service.
-   *      holdingUpdated is invoked when the initial holdings are read from the chain.
-   *  2. The second event is triggered by the server wallet deposit. The deposit results in a holdingUpdated invocation.
-   *  3. The third event is triggered by the browser wallet deposit.
-   */
-
```

Comment:
This removed comment could maybe be supplied with the `containsPostfundState` function?

Reply:
See https://github.com/statechannels/statechannels/pull/3515#discussion_r616954396 for why this comment is no longer relevant. We are no longer waiting for the 3rd notification from the wallet. Instead, we are waiting for a notification with turn number 3. These are actually very different.